### PR TITLE
docs/rancher *: Add docs about deploy radondb mysql on rancher #338

### LINKED ISSUE #338: documentation：Add docs about deploying radondb-mysql-operator on container-management-platform
**Is your feature request related to a problem? Please describe.**

[N/A]

**Describe the solution you'd like**

[N/A]

**Describe alternatives you've considered**

[N/A]

**Additional context**

[N/A]

### PR DESCRIPTION
**Pull request by molliezhang**
### What type of PR is this?

/documentation
/enhancement

### Which issue(s) this PR fixes?

Fixes #338 

### What this PR does?

Summary:

### Special notes for your reviewer?
@andyli029 @runkecheng @KID-G @qianfen2021 @acekingke

### CODE REVIEW

File: docs/rancher/deploy_radondb-mysql_operator_on_rancher.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+   正常连接数据库回显信息如下图：
+
+   ![Pod Running](_images/pod_running.png)
+
+## 访问 RadonDB MySQL
+
+您需要准备一个用于连接 MySQL 的客户端。
+
+### 同 NameSpace 访问
```
Comment (runkecheng) [inline docs/rancher/deploy_radondb-mysql_operator_on_rancher.md:147]: Here can be simplify.

File: docs/rancher/deploy_radondb-mysql_operator_on_rancher.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+   mysqlcluster.mysql.radondb.com/sample created
+   ```
+
+#### 通过导入 YAML 方式
+
+1. 下载 [RadonDB MySQL 集群配置样例](/config/samples/mysql_v1alpha1_mysqlcluster.yaml)，修改 YAML 文件中配置参数值。
+
+   详细配置参数说明，请参见[配置](#配置)。
+
+2. 在 Rancher 集群管理页面，点击右上角 YAML 导入图标。自啊弹出的窗口中，导入修改后的 YAML 文件。
```
Comment (runkecheng) [inline docs/rancher/deploy_radondb-mysql_operator_on_rancher.md:117]: 自动弹出

File: docs/rancher/deploy_radondb-mysql_operator_on_rancher.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+> 
+> 准备可用于连接 MySQL 的客户端。
+
+- 当客户端与数据库部署在不同 Rancher 集群，请[在 Rancher 中设置 Load Balancer 和 Ingress Controller](https://rancher.com/docs/rancher/v2.6/en/k8s-in-rancher/load-balancers-and-ingress/)。
+  
+  更多访问不同集群中数据库方式说明，请参见 [Kubernetes 访问集群中的应用程序](https://kubernetes.io/zh/docs/tasks/access-application-cluster/)。
+
+- 当客户端与数据库部署在同一 Rancher 集群内，可选择使用 `service_name` 或者 `cluster_IP` 方式，访问 RadonDB MySQL。
+
+以下主要演示客户端与数据库在同一 Rancher 集群内场景，访问 RadonDB MySQL 的方式。
```
Comment (runkecheng) [inline docs/rancher/deploy_radondb-mysql_operator_on_rancher.md:156]: 以下主要演示在...的场景下

File: docs/rancher/deploy_radondb-mysql_operator_on_rancher.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+    $ mysql -h sample-follower.default -u radondb_usr -p  
+    ```
+
+### `cluster_IP` 方式
+
+```shell
+mysql -h <cluster_IP> -P <mysql_Port> -u <user_name> -p
+```
+
+以下示例用户名为 `radondb_usr`，cluster IP 为 `10.10.128.136` ，连接示例如下：
```
Comment (runkecheng) [inline docs/rancher/deploy_radondb-mysql_operator_on_rancher.md:190]: 可以用主/从服务的cluster IP来举例

**Review by runkecheng — CHANGES_REQUESTED** [review]

**Review by runkecheng — APPROVED** [review]
lgtm

**Review by runkecheng — APPROVED** [review]

**Comment by andyli029** [comment]
LGTM.

**Pull request merged**